matchmaker/views/external_api.py

```diff
@@ -1,4 +1,11 @@
+from __future__ import unicode_literals
+
 import json
+try:
+    from json import JSONDecodeError
+except Exception:
+    JSONDecodeError = Exception
```

Comment:
In redis_utils you just se `ValueError` for both python versions - why does that not work here?

Reply:
I thought `JSONDecodeError` is more specific for json decoding. `ValueError` also works but it is not for json only. How about changing it to as below?
```
try:
    from json import JSONDecodeError
except Exception:
    JSONDecodeError = ValueError
```

Reply:
No there are `ValueError` that have other messages as well. I think if `JSONDecodeError` isn't defined you should just define it as new Exception class, and then it won't ever get raised but thats fine because the python 2 error handling will stay the same. Also, `hast=attr` is a way better way to check if something is defined than try-except

Reply:
Updated. Please review.